helloworld/H7_E1.c

```diff
@@ -0,0 +1,71 @@
+#include <stdio.h>
+
+int main() {
+	//1.-2. feladat
+	//What is the difference between 
+	//the following two pieces of code?
+	// for (i=0, j=1; i<8; i++, j++)
+	// printf("%d  +  %d  =  %d\n", i, j, i+j);
+	//
+	//for (i=0, j=1; i<8; i++, j++);
+	//printf("%d  +  %d  =  %d\n", i, j, i+j);
+	
+	//in the line 11 there is an "empty" loop, it does not anyting 
+	
+	//2. feladat
+	//Write a program that contains the two pieces of 
+	//code shown in exercise 1, and then execute the program. 
+	//What are you going to see on the screen?
+	int i;
+	int j;
+	printf("2. feladat\n");
+	for (i = 0, j = 1; i < 8; i++, j++)
```

Comment:
akkor is legyen kinn a kapcsos zárójel a for loopnál, ha csak egy sorból áll a teste. Ugyanez igaz az if-re, a while-ra, stb. Vagy itt pont ez volt a feladat?